apis/beacon/pool/validator_registration.yaml

```diff
@@ -0,0 +1,21 @@
+get:
+  operationId: getValidatorRegistrationV1
```

Comment:
registrations rather than registration

used ValidatorRegistrationV1 since it's the exact spec name